modules/challenges/LuckyNumbersModule.js

```
@@ -0,0 +1,47 @@
+
+/**
+ * Generates an error message based on the validity of the input.
+ *
+ * @param {string|null|undefined} inputValue - The input value to validate.
+ * @returns {string} An error message if the input is invalid, otherwise an empty string.
+ */
+export function errorMessage(inputValue) {
+  if (inputValue === null || inputValue === undefined || inputValue.trim() === '') {
+    return 'Required field';
+  }
+  return !isNaN(Number(inputValue)) && Number(inputValue) !== 0 ? '' : 'Must be a number besides 0';
```

Biome (1.9.4)

[error] isNaN is unsafe. It attempts a type coercion. Use Number.isNaN instead.

See the MDN documentation for more details.
Unsafe fix: Use Number.isNaN instead.


(lint/suspicious/noGlobalIsNan)